app/client/components/SpeedIndicator.js

```diff
@@ -27,7 +33,7 @@ export default class SpeedIndicator extends React.Component {
         }
 
         return (
-            <div style={ speedStyle }>
+            <div style={ speedStyle } onClick={(e) => {this.speedClicked(e)}}>
```

Comment:
never use `this.methodName`, it is more explicit if you import the symbol into the method lie you do with props:

```
var { speedClicked } = this;
```

also you don't need the arrow function if you want the callbac to receive all the arguments that are given by the event:

```
onClick={speedClicked}
```